docs/reference-metrics-opentelemetry.md

```diff
@@ -0,0 +1,22 @@
+---
+id: reference-metrics-opentelemetry
+title: Pulsar OpenTelemetry Metrics
+sidebar_label: "OpenTelemetry Metrics"
+---
+
+Pulsar exposes the following OpenTelemetry metrics.
+
+## Broker
+
+### Topic Lookup metrics
+
+| Name                                                | Type              | Unit          | Attributes                        | Description                                                                                                                                                             |
+|-----------------------------------------------------|-------------------|---------------|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| pulsar.broker.lookup.request.duration               | Histogram         | `second`      |                                   | The duration of topic lookup requests (either binary or HTTP)                                                                                                           |
```

Comment:
I have a problem with this as a whole.
I find the following hard to digest:
<img width="1056" alt="image" src="https://github.com/apache/pulsar-site/assets/989425/1bfeccfb-8358-4fdd-a151-34337fa5cc92">

How about we go without tables?

<img width="1005" alt="image" src="https://github.com/apache/pulsar-site/assets/989425/054fc392-03c4-4000-95c4-8032b027aea1">

Markdown:
```
# Broker

## Topic Lookup metrics

### pulsar.broker.lookup.request.duration
The duration of topic lookup requests (either binary or HTTP)
* Type: Histogram
* Unit: `second`
* Attributes:
  * `pulsar.lookup.response` - The response type of the lookup request
    * `failure`
    * `broker`
    * `redirect`

### pulsar.broker.request.topic.lookup.concurrent.usage
The number of pending lookup operations in the broker. When it reaches threshold "maxConcurrentLookupRequest" defined in broker.conf, new requests are rejected.
* Type: UpDownCounter
* Unit: `{operation}`

```

Reply:
That's one of my concerns here. Tables don't render great with so many columns, but listing the metrics one by one would yield a very large document. Perhaps split the files by metric domain?